nuve/nuveAPI/resource/tokensResource.js

```diff
@@ -104,7 +104,8 @@ var generateToken = function (callback) {
 
             token = currentService.testToken;
 
-            log.info('TestToken already exists, sending it', token);
+            log.info('message: generateTestToken already generated - returning, ' + 
```

Comment:
add a `+` to the end of the line
`log.info('message: generateTestToken already generated - returning, ' + `